tock-process-console/.gitignore

```diff
@@ -0,0 +1,6 @@
+/target
+/board-mock.log
+/board-mockk.log
+/loguri.log
+/src/.DS_Store
+/src/ui_management/.DS_Store
```

Comment:
instead of adding each individual file, add a glob:
```rust
*.log
.DS_Store
```
Notice the lack `/` at the start. This means any files that ends in .log or any file that is ".DS_Store" will be automatically ignored